packages/tools/src/tools/annotation/AngleTool.ts

```diff
@@ -241,9 +242,8 @@ class AngleTool extends AnnotationTool {
   };
 
   toolSelectedCallback = (
-    evt: EventTypes.MouseDownEventType,
-    annotation: AngleAnnotation,
-    interactionType: InteractionTypes
+    evt: EventTypes.InteractionEventType,
```

Comment:
I don't understand why our `MouseDownEventType` is changed to `InteractionEventType`? I expected it be `InteractionDownEventType` or similar

Reply:
Since the function doesn't use anything specific to the `DownEventType` we adopt the generic `InteractionEventType`